src/Angor/Client/Pages/Signatures.razor

```diff
@@ -132,6 +134,7 @@
                                 <th class="text-uppercase text-xxs font-weight-bolder opacity-7">Investment</th>
                                 <th class="text-uppercase text-xxs font-weight-bolder opacity-7">Received</th>
                                 <th class="text-uppercase text-xxs font-weight-bolder opacity-7">Status</th>
+                                <th class="text-uppercase text-xxs font-weight-bolder opacity-7">Chat</th>
```

Comment:
Mixing two things in one pr is not good, the chat pr is a separate pr

Reply:
Sorry, I thought I was working on a new branch.